src/Storages/ObjectStorageQueue/ObjectStorageQueueOrderedFileMetadata.cpp

```diff
@@ -523,7 +530,7 @@ void ObjectStorageQueueOrderedFileMetadata::migrateToBuckets(const std::string &
         {
             throw Exception(ErrorCodes::SUPPORT_IS_DISABLED,
                             "Cannot change old metadata structure, "
-                            "because someone is modifying it concurrently");
+                            "because someone is modifying it concurrently by another replica");
```

Comment:
`nit:`
```suggestion
                            "because someone is modifying it concurrently from another replica");
```

Reply:
Missed the line, should be like this [03d28ff](https://github.com/ClickHouse/ClickHouse/pull/77448/commits/03d28ff5ddf66d97fe51caa65230b0d77a7b37a0)